affine_transfo.py

```diff
@@ -50,46 +49,50 @@ def get_parser():
         help='Help',
         nargs="*"
     )
+    optional.add_argument(
+        '-o',
+        help='Output filename extension to identify new file,\nexample: (input = t) sub-amu01_T2W.nii.gz ==> sub-amu01_T2w_t.nii.gz',
```

Comment:
extension is what comes after ".". Here you're talking about a suffix
also: i recommend against adding a "_" without the user specifying it (what if users prefer "-"?). 
```suggestion
        help="Suffix for output file name.\nexample: '-i MYFILE.nii -o _t' would output MYFILE_t.nii",
```